hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -58,6 +58,10 @@
   private static final String DEFAULT_ASSUME_DATE_PARTITIONING = "false";
   private static final String HOODIE_WRITE_STATUS_CLASS_PROP = "hoodie.writestatus.class";
   private static final String DEFAULT_HOODIE_WRITE_STATUS_CLASS = WriteStatus.class.getName();
+  private static final String HOODIE_FINALIZE_WRITE_BEFORE_COMMIT = "hoodie.finalize.write.before.commit";
```

Comment:
we always call `finalizeWrite()` api method. so this is a little confusing. 

Please name these configs ` hoodie.copyonwrite.use.temp.folder`  & ` hoodie.finalize.parallelism` and the variables `HOODIE_COPYONWRITE_USE_TEMP_FOLDER` so on..